composer.json

```
@@ -364,7 +364,8 @@
                 "Statically cache editor attachments per format (https://www.drupal.org/i/3447794)": "patches/3447794-mr-9609.patch",
                 "An update to symfony/http-foundation plus a trailing space took down the views UI (https://www.drupal.org/project/drupal/issues/3486195)": "https://www.drupal.org/files/issues/2024-11-27/3486195-46.patch",
                 "Error: Cannot read properties of undefined (reading 'settings')\u2028 with dialog.position.js (https://www.drupal.org/project/drupal/issues/3356667)": "https://www.drupal.org/files/issues/2024-09-19/3356667-69-drupal-core-10.3.5-undefined.patch",
-                "CKEditor 5 balloons invisible when CKEditor 5 is used inside a modal (https://www.drupal.org/project/drupal/issues/3328425)": "https://www.drupal.org/files/issues/2024-09-19/3328425-50-drupal-core-10.3.5-ckeditor5-ballon.patch"
+                "CKEditor 5 balloons invisible when CKEditor 5 is used inside a modal (https://www.drupal.org/project/drupal/issues/3328425)": "https://www.drupal.org/files/issues/2024-09-19/3328425-50-drupal-core-10.3.5-ckeditor5-ballon.patch",
+                "Display Bug when using #states (Forms API) with Ajax Request": "patches/core-1091852-186-display-bug-when-using-states-with-ajax-request.patch"
```

Currently for patches that we are using directly from drupal.org comments, we typically load directly from drupal.org. The following suggestion will do this, and then you can delete the patch file: 
```suggestion
                "Display Bug when using #states (Forms API) with Ajax Request (https://www.drupal.org/project/drupal/issues/1091852)": "https://www.drupal.org/files/issues/2023-12-18/1091852-186.patch"
```